### DISCUSSION TITLE
Session management for Supabase auth

### DISCUSSION THREAD

**Discussion by CharlesFJWilliamson** [supabase/supabase#8535] [Questions] [answered]
I've been looking into Supabase Auth, but I've found there isn't much information within the documentation about how sessions are managed. In particular, I'm interested in knowing whether there is a way to configure the expiration time of a token (or to set it to not expire by default), as well as being able to allow users to log themselves out based on devices that they've previously logged in with. Is any of this possible with auth?

**Comment by GaryAustin1** [comment, marked as answer]
You can set jwt token expiration time in the database dashboard auth settings. I do not know the longest time it can be set to. 
If you use the signOut call all devices will lose their refresh tokens, including the current one.  But the jwts are good until they expire, they just can’t be refreshed.

**Reply by CharlesFJWilliamson** [reply]
Thank you! I'd missed that option in settings

**Reply by CharlesFJWilliamson** [reply]
Seems like the maximum is still only a week, but that's still useful
